app/models/chargeback_rate_detail.rb

```diff
@@ -36,9 +67,12 @@ def find_rate(value)
 
   def cost(value)
     return 0.0 unless self.enabled?
-    value = 1 if group == 'fixed'
+
+    value = 1.0 if fixed?
+
     (fixed_rate, variable_rate) = find_rate(value)
-    hourly(fixed_rate) + hourly(variable_rate) * value
+
+    hourly(fixed_rate) +  hourly(variable_rate) * value
```

Comment:
@gtanzillo Nice finding, but I need to confirm these calculation which was used for it:
There are used this calculations:

I guess that was used this rate for Storage Allocate Cost:
**Fixed:** $1
**Variable:** $0
**Per Time:** Hourly
**Per Unit:** GB

**Report setttings:** Monthly

I guess fixed part should not be depend on metric value (value of Storage Allocate):
Hourly rate for GB: $1 (we have it already in hours)
Hourly rate for B(bytes, it is unit in MetricRollup table):` $1 / (1024)^3 = 9.313225746154785e-10`

9.313225746154785e-10 * 744(hours in month)  = `6.92903995513916e-07`

and in report it was rounded to $0.

Can you confirm it please ?